docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
 title: How To Show and Hide a Split View Pane with MVVM
 ---
 
-import SplitViewExpanderScreenshot from '/img/reference/controls/splitview/splitview-expander.gif';
+import FirstExample from '/img/guides/development-guides/How-to-show-and-hide-a-split-view-pane-with-MVVM/First-example.PNG';
+import SecondExample from '/img/guides/development-guides/How-to-show-and-hide-a-split-view-pane-with-MVVM/Second-example.PNG';
+import PaneHidden from '/img/guides/development-guides/How-to-show-and-hide-a-split-view-pane-with-MVVM/Pane-Hidden.PNG';
+import PaneShown from '/img/guides/development-guides/How-to-show-and-hide-a-split-view-pane-with-MVVM/Pane-shown.PNG';
 
-Content in preparation.
+# How To Show and Hide a Split View Pane with MVVM
+In this guide you will learn how to show and hide a split view panel using the MVVM pattern. If you are new to Avalonia UI please check out the following documents: [The MVVM patter](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/) - [Avalonia UI and MVVM](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/avalonia-ui-and-mvvm).
+To proceed in this guide you will need at least a basic understanding of what Data Binding is and how it works in Avalonia.
+If you are coming from WPF, then the mechanism is nearly the same, except for some differences that will not be covered in this guide.
 
-You can use the MVVM pattern with the split view control to implement a 'tool pane' style UI.
+## Starting
+First of all, you need to create an MVVM application project, which you can do by reading the following guide [Create and run a project](https://docs.avaloniaui.net/docs/get-started/test-drive/create-a-project).<br>
+Once you created the project you should be presented with the following files:
+- MainWindow.axaml
+- MainWindowViewModel.cs
 
+Those are the files that you will be editing, except for the one that you will be creating during this guide.
+
+:::info
+Although here I will be using the MainWindow.axaml and MainWindowViewModel.cs files, this guide will work for every other View and its respective ViewModel.
```

Comment:
I meant Views and ViewModels that someone may have/create in the app. I edited it so that it specify.
`Although in this guide we will be using the MainWindow.axaml and MainWindowViewModel.cs files, this guide will work for every additional View and ViewModel you may create.`